src/settings/PreferencesSettingsTab.tsx

```diff
@@ -21,11 +23,17 @@ export const PreferencesSettingsTab: FC = () => {
     showHandRaisedTimerSetting,
   );
 
+  const [showReactions, setShowReactions] = useSetting(showReactionsSetting);
+
+  const [playReactionsSound, setPlayReactionSound] = useSetting(
+    playReactionsSoundSetting,
+  );
+
   const onChangeSetting = useCallback(
```

Comment:
The point of `useCallback` (referential stability) is kind of defeated if you're passing a new function on each render with `(e) => onChangeSetting(...)` below :) not that it matters much

Reply:
oh yeah...duh